packages/ui/src/2_molecules/WalletContainer/WalletContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import { Icon } from '../../1_atoms/Icon/Icon';
+import styles from './WalletContainer.module.css';
+
+type WalletContainerProps = {
+  name: string;
+  icon?: ReactNode;
+  tooltip?: string;
+  className?: string;
+  dataLayoutId?: string;
+};
+
+export const WalletContainer: FC<WalletContainerProps> = ({
+  name,
+  icon,
+  tooltip,
+  className,
+  dataLayoutId,
+}) => {
+  return (
+    <div
+      data-layout-id={dataLayoutId}
+      className={classNames(className, styles.walletContainer)}
+    >
+      <div className="flex items-center justify-center">
```

Comment:
move tailwind classes to css module (build will not work with inline tailwind classes)

```suggestion
      <div className={styles.label}>
```